openapi-cli/src/test/java/io/ballerina/openapi/generators/schema/FieldGenWithNullableOption.java

```diff
@@ -0,0 +1,82 @@
+package io.ballerina.openapi.generators.schema;
```

Comment:
Please add licence header in the top of the file.
In any java file or bal file, we should add licence header in the top.